packages/selleo-design-core/src/Icon.tsx

```diff
@@ -0,0 +1,14 @@
+import { h } from 'preact';
+
+import iconsData from '../../../apps/docs/src/components/Icons/data';
```

Comment:
that's a cross resource. We shouldn't be importing data from apps into packages, only the other way around

Reply:
Good to know, done :)